dto/src/test/java/ca/on/oicr/gsi/runscanner/rs/dto/test/dragen/JacksonTest.java

```diff
@@ -0,0 +1,189 @@
+package ca.on.oicr.gsi.runscanner.rs.dto.test.dragen;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import ca.on.oicr.gsi.runscanner.dto.dragen.AnalysisFile;
+import ca.on.oicr.gsi.runscanner.dto.dragen.DragenAnalysisUnit;
+import ca.on.oicr.gsi.runscanner.dto.dragen.DragenPipelineRun;
+import ca.on.oicr.gsi.runscanner.dto.dragen.DragenWorkflowRun;
+import ca.on.oicr.gsi.runscanner.dto.dragen.samplesheet.Samplesheet;
+import ca.on.oicr.gsi.runscanner.dto.dragen.samplesheet.SamplesheetBCLConvertSection;
+import ca.on.oicr.gsi.runscanner.dto.dragen.samplesheet.SamplesheetBCLConvertSection.SamplesheetBCLConvertDataEntry;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.nio.file.Path;
+import java.time.Instant;
+import org.junit.jupiter.api.Test;
+
+// TODO this isn't running on build
```

Comment:
I thought, 'maybe an older version of surefire doesn't look in subdirectories'. So I tried upgrading surefire to 3.5.2. That version _only_ runs JacksonTest and skips the NotificationDto tests one level up. ???